pkg/server/server.go

```diff
@@ -194,24 +195,69 @@ func coreControllers(ctx context.Context, sc *Context, config *Config) error {
 
 func stageFiles(ctx context.Context, sc *Context, controlConfig *config.Control) error {
 	dataDir := filepath.Join(controlConfig.DataDir, "static")
-	if err := static.Stage(dataDir); err != nil {
+	stageTraefik := doStageTraefik(sc)
+	if err := static.Stage(dataDir, stageTraefik); err != nil {
```

Comment:
I suggest no need to add this specific behavior for the needless change because these files saved just for static files. The important part is what you have done for the manifest part.